bridges/EconomistBridge.php

```diff
@@ -0,0 +1,64 @@
+<?php
+class EconomistBridge extends BridgeAbstract {
+	const NAME = 'The Economist: Latest Updates';
+	const URI = 'https://www.economist.com';
+	const DESCRIPTION = 'Fetches the latest updates from the Economist.';
+	const MAINTAINER = 'thefranke';
+	const CACHE_TIMEOUT = 60 * 60; // 1h
+
+	public function collectData() {
+		$html = getSimpleHTMLDOM(self::URI . '/latest/')
+			or returnServerError('Could not fetch latest updates form The Economist.');
+
+		$limit = 0;
+
+		foreach($html->find('article') as $element) {
+			if($limit >= 10)
+				break;
+
+			$a = $element->find('a', 0);
+			$href = self::URI . $a->href;
+			$full = getSimpleHTMLDOMCached($href);
+			$article = $full->find('article', 0);
+
+			$header = $article->find('h1', 0);
+			$author = $article->find('span[itemprop="author"]', 0);
+			$time = $article->find('time[itemprop="dateCreated"]', 0);
+			$content = $article->find('div[itemprop="description"]', 0);
+
+			// Remove newsletter subscription box
+			$newsletter = $content->find('div[class="newsletter-form__message"]', 0);
+			if ($newsletter)
+				$newsletter->outertext = '';
+
+			$newsletterForm = $content->find('form', 0);
+			if ($newsletterForm)
+				$newsletterForm->outertext = '';
+
+			// Remove next and previous article URLs at the bottom
+			$nextprev = $content->find('div[class="blog-post__next-previous-wrapper"]', 0);
+			if ($nextprev)
+				$nextprev->outertext = '';
+
+			$full->save();
```

Comment:
What is the purpose of this call?
The `save` returns a string, but it is currently not assigned to any value.

Reply:
No purpose, I used it to debug and forgot to take it out.